cli/create/templates/templates.go

```diff
@@ -85,6 +89,42 @@ func Instantiate(ctx *context.Ctx) error {
 	return nil
 }
 
+func parseStaticTemplate(fs http.FileSystem) (*templates.FileTreeTemplate, error) {
+	var tmpl templates.FileTreeTemplate
+
+	err := vfsutil.Walk(fs, "/", func(filePath string, fileInfo os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if fileInfo.IsDir() {
+			tmpl.AddDirs(templates.DirTemplate{
+				Path: filePath,
+				Mode: fileInfo.Mode(),
+			})
+		} else {
+			fileContent, err := getStaticFileContent(fs, filePath)
+			if err != nil {
+				return fmt.Errorf("Failed to get file content: %s", err)
+			}
+
+			tmpl.AddFiles(templates.FileTemplate{
+				Path:    filePath,
+				Mode:    0755,
```

Comment:
Is it possible to collect file modes on generating static filesystem and then use them here?

I mean smth like `fileModeByPaths["<file-path>"] = <file-mode>`.